tests/stdcxx/enc/enc.cpp

```diff
@@ -241,6 +241,6 @@ OE_SET_ENCLAVE_SGX(
     1,    /* ProductID */
     1,    /* SecurityVersion */
     true, /* Debug */
-    512,  /* NumHeapPages */
-    512,  /* NumStackPages */
+    1024, /* NumHeapPages */
+    1024, /* NumStackPages */
```

Comment:
Thanks for all the work. Is there something that can be put in release notes about developers needing to increase EPC(and by how much) in config with the latest snmalloc? That way, people are not caught unaware.

Reply:
@radhikaj in general 0.6.0 should use approximately the same amount of memory.  

The key change that is affecting this for OE is that we now adaptively grow the memory usage, so that initially it can work on small footprints, but scale to larger footprints with high core counts.  As memory is requested it reserves larger and larger chunks up to some limit.  Larger chunks reduce contention and improve throughput. Effectively, a thread will grow its memory usage by:
 16KiB, 16KiB, 32KiB, 64KiB, ... 1MiB, 2MiB, 2MiB, ...

I imagine on the two tests that @achamayou had to increase the footprint for the heuristic pulled in a larger chunk, and then most wasn't used.  But growing like this should make running on the 48 core Icelake SKU much better. 

FYI, I added this just for OE ;)  https://github.com/microsoft/snmalloc/pull/511

Reply:
@radhikaj the vast majority of the tests did not require adjustment, and the two that did were done quite coarsely because the binary search to figure out the minimum change is time consuming. I've tweaked them now to minimise the values, but as explained by @mjp41 the new heuristic will sometimes result in a small increase in memory usage in return for decreased contention.